app/views/layouts/application.html.erb

```diff
@@ -0,0 +1,39 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <title>2Save</title>
+    <meta name="viewport" content="width=device-width,initial-scale=1">
+    <%= csrf_meta_tags %>
+    <%= csp_meta_tag %>
+
+    <%= stylesheet_link_tag "application", "https://fonts.googleapis.com/css2?family=Montserrat:wght@500;700&display=swap", "data-turbo-track": "reload" %>
+    <%= javascript_importmap_tags %>
+  </head>
+
+  <body>
+  <%if user_signed_in? %>
+  <nav class="nav section-name form-nav">
+       <% if current_page?(root_path) || current_page?(controller: "groups", action: "new") %>
+      <div class="new-my-form">
+        <%= image_tag("https://img.icons8.com/ios-glyphs/30/FFFFFF/menu--v1.png") %>
+        <h2>CATEGORIES</h2>
+        <%if user_signed_in? %>
+        <%= button_to "Sign Out", destroy_user_session_path, class: "signout", method: :delete %>
+        <% end %>
+      </div>
+      <% else %>
+      <div class="new-my-form">
+        <%= link_to '<', root_path %>
```

Comment:
- [ ] This "<" button should work differently for different pages. When I am on the page for adding an expense (transaction) and I click this button I should be taken to the transaction page for that category. Kindly implement that as well.

Reply:
Done 👍🏼  @TedLivist